stacks/jenkins-scale.yml

```diff
@@ -24,6 +24,7 @@ services:
           memory: 800M
     extra_hosts:
       - "${SLACK_HOST:-devops20.slack.com}:${SLACK_IP:-54.192.78.227}"
+      - "${GITHUB_HOST:-github.com}:${GITHUB_IP:-54.72.52.58}"
```

Comment:
I think we need to remove this one, mostly because the docs and the books that use this file do not yet have the instructions and the IP already changed (now it's 140.82.118.3). It might fail without people knowing why.

How about moving this to a separate file, and updating the docs later on to use that one instead?

Reply:
Roger